packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1759,8 +1759,10 @@ export default class BackupNg {
       await Promise.all(
         files.filter(isMetadataFile).map(async file => {
           const path = `${dir}/${file}`
+          const backupFile = file.split('.')[0]
           try {
             const metadata = JSON.parse(String(await handler.readFile(path)))
+            metadata.size = await handler._getSize(`/${dir}/${backupFile}.xva`)
```

Comment:
It only handles backups of type `full`. Backups of type `delta` have files with extension `vhd`.

Do you want to dedicate this PR for full backups? And create another PR for delta backups?

Reply:
As you want. I think we need to do both in this PR no ? @julien-f

Reply:
It's up to you, you can either do them in different PRs or in the same one.

IMHO, it's a bit different for deltas and there are some questions to answer, so it might be a good idea to do them in another PR.